### PR TITLE
[FIX]Fixed the issue of expand.

### PR DESCRIPTION
**Pull request by drashtipatel24**
**Fixed issued**

1. Fixed issued of single view not render a button. #391 

2.  expand button to display in another views.
     1. Go to list view.
      2. added group-by
      3. Click on kanban view or any other view.
      4. multiple time render views it displays expand button.

3. Improved code for a module.
    1. Many methods directly overwrite without super so we improved this method.
    2. used super called and removed unused method.

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
@parvezqureshi review!

**Comment by parvezqureshi** [comment]
I reviewed code,  @drashtipatel24 please check following improvement:

1)Remove unnecessary import js file and variable
    1.1)var config = require('web.config')
    1.2)var HEADING_COLUMNS_TO_SKIP_IN_GROUPS = 2
    1.3)var _t = core._t;
2)Please Use "use strict";
3)Please Set import js file in alphabetical order.

**Comment by JayVora-SerpentCS** [comment]
code line to code line karo review guru!

File: web_groupby_expand/static/src/js/web_groups_expand.js [hunk 1]
```diff
@@ -1,26 +1,77 @@
...
-        start: function() {
-            this._super.apply(this, arguments)
-            var self = this;
-            var is_grouped = !!self.renderer.state.groupedBy.length;
-            var oe_list_expand = $("#expand_icon");
-            _.each(this.controlPanelElements.$switch_buttons, function(rec){
-                if (rec.id == 'expand_icon'){
-                    oe_list_expand = $(rec);
-                }
+var HEADING_COLUMNS_TO_SKIP_IN_GROUPS = 2
```
Comment (parvezqureshi) [inline web_groupby_expand/static/src/js/web_groups_expand.js]: Remove unnecessary variable.

File: web_groupby_expand/static/src/js/web_groups_expand.js [hunk 2]
```diff
@@ -1,26 +1,77 @@
 odoo.define('web_groupby_expand.web_groupby_expand', function(require) {
```
Comment (parvezqureshi) [inline web_groupby_expand/static/src/js/web_groups_expand.js:3]: Please Use "use strict";

File: web_groupby_expand/static/src/js/web_groups_expand.js [hunk 3]
```diff
@@ -1,26 +1,77 @@
...
 
-    var core = require('web.core');
-    var ListRenderer = require('web.ListRenderer')
-    var ListController = require('web.ListController')
-    var AbstractController = require('web.AbstractController');
-    var _t = core._t;
+var core = require('web.core');
+var ListRenderer = require('web.ListRenderer')
+var ListController = require('web.ListController')
+var AbstractController = require('web.AbstractController');
```
Comment (parvezqureshi) [inline web_groupby_expand/static/src/js/web_groups_expand.js]: Please Set import js file in alphabetical order.

**Review by parvezqureshi — CHANGES_REQUESTED** [review]

**Comment by parvezqureshi** [comment]
**Code Reviewed.**
Tested in local working correctly.

**Comment by Kontrolix** [comment]
I test it and it does the job for me.

**Pull request merged**